index.html

```diff
@@ -1,62 +1,101 @@
     <h2 id = "text">About:</h2>
-    <img src ="img/OksanaF.jpg" alt ="Oksana in green dress">
-        <p>Hi I'm Oksana. <em>I really love programming and creating websites</em> It is a pleasure for me to create projects for you. 
-        My passion is listening to people and working with them to convert their plans and ideas to reality.</p>
-        <p>Also, I put my attention into every single detail, and making your dreams come true!<br> Knowing the principles of UI/UX is helping me to create user friendly designs for my coding projects.</p>
+    <img src ="img/OksanaF.jpg" alt ="Oksana in a green dress">
+        <p>Hi I'm Oksana. <em>I really love programming and creating websites</em>. It is a pleasure for me to create projects for you. </p>
+        <p>My passion is listening to people and working with them to convert their plans and ideas to reality.</p>
+        <p>Also, I put my attention into every single detail, and making your dreams about websites come true!<br></p>
+        <p> Knowing the principles of UI/UX is helping me to create user friendly designs for my coding projects.</p>
         <p> My family moved to the USA 13 years ago from Uzbekistan and I admire this country for all the opportunities I have gotten. </p>
         <p>I live in the Research Triangle Park area of North Carolina.  When I'm not on my computer, you'll find me playing with my kids, walking my beagle or exploring the area.</p>
 </section>
 
+<!---<section>  First version of Experience with ul and li  
+    <h2 id = "text">Experience:</h2>
+    <div class="grid-container">
+        <ul>
+            <li id ="experience">
+                <span><strong>Wake Tech. Community College:</strong> AAS Major Web Technologies/ Web Design</span>
+                <span class = "years">2014- 2018</span>  
+            </li>
+            <li id ="experience">
+                <span><strong>Red Rover Bootcamp:</strong></span> 
+                <span class = "years">March 2023- May 2023</span>
+            </li>
+            <li id ="experience">
+                <span><strong>Code The Dream:</strong></span>
+                <span class = "years"> April 2023- current</span>
+            </li>
+        </ul>
+    </div>
+</section>
+--->
+<!--2nd version of experience using grid-->
 <section>
-    <h3 id = "text">Experience:</h3>
-    <ul>
-        <li id ="experience">Wake Tech. Community College AAS Major Web Technologies/ Web Design: 2014- 2018 </li>
-        <li id ="experience">Red Rover Bootcamp: March 2023- May 2023</li>
-        <li id ="experience">Code The Dream: April 2023- current</li>
-    </ul>
+    <h2 id = "text">Experience:</h2>
+    <div class="grid-container">
+        <div id = "experience">Code The Dream: April 2023- current</div>
+        <div id = "experience">Red Rover Bootcamp:March 2023- May 2023</div>
+        <div id = "experience">Wake Tech. Community College: AAS Major Web Technologies/ Web Design: 2014- 2018</div>
+    </div>
```

Comment:
Nitpick: Same feedback as the above about using ids with the same name. But, since these `div` elements are just containing text, it's recommended that they be changed to a semantic HTML element (such as a `p` tag since it's like a paragraph of text). You can read more about semantic HTML [here](https://www.w3schools.com/html/html5_semantic_elements.asp).